install/helm/gloo/generate.go

```diff
@@ -107,7 +107,7 @@ func generateChartYaml(version string) error {
 		return err
 	}
 
-	chart.Version = "1.9.26-OPENET"
+	chart.Version = "1.9.30-OPENET-SNAPSHOT"
```

Comment:
Hi @prasanth-openet . All good, I think now you should hardcode the version to 1.9.30-OPENET, and then you can merge it to the target branch. Then, create a release version with 1.9.30-OPENET

Reply:
I have removed it now.